src/lib.rs

```diff
@@ -25,11 +25,9 @@ use std::io::prelude::*;
 use std::io::{self, Error, ErrorKind, SeekFrom};
 use std::iter::repeat;
 use std::mem;
+use std::path::{Path, PathBuf};
 use std::str;
 
-#[cfg(feature = "nightly")]
```

Comment:
I think for now it may not be worth trying to get `unpack` working on Nightly due to the Unix/Windows requirements